src/Components/Map.js

```diff
@@ -0,0 +1,106 @@
+import React, { useState } from 'react';
+import Tile from './Tile.js';
+import Player from './Player.js';
+import '../Styles/Map.css';
+
+const Map = () => {
+  const [tileGrid, setTileGrid] = useState([]);
+  const tileNumbers = [];
+  for (let i = 0; i < 225; i++) {
```

Comment:
I think we should probably put this logic in to useEffect to only run one time, then layer on other components so we don't have to update state and re-render everything whenever a player moves.  We might even be able to store this logic in it's own component or file and have useEffect run it, then just have the data we need.  It might make this a little easier to look at.